ui/views.py

```diff
@@ -136,6 +137,16 @@ def __call__(self, request, repo_slug):
         if repo_slug not in set([x.slug for x in repos]):
             return HttpResponseForbidden("unauthorized")
         self.repo = [x for x in repos if x.slug == repo_slug][0]
+        # get sorting from params if it's there
+        params = dict(request.GET.copy())
+        if ('sortby' in params and
```

Comment:
It can be done: just checked and `request.GET` has a method `pop`.
It still feels weird to me to modify the request object, but I have no strong opinions about this.

Anyway, I agree that my code could be better and I modified it to:
```
# get sorting from params if it's there
        sortby = dict(request.GET.copy()).get('sortby', [])
        if (len(sortby) > 0 and
                sortby[0] in LoreSortingFields.all_sorting_fields()):
            self.sortby = sortby[0]
        else:
            # default value
            self.sortby = LoreSortingFields.DEFAUT_SORTING_FIELD
```